engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -92,6 +95,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   }
 
   // When PreparingJob, the FSM always has NoData
+  //RUCHI: Note, the backend preparation actor failed to intialize a few times, make a ticket or figure out whats up
```

Comment:
I'll remove the comment and just make a ticket if I see it furthermore in my testing :)